### DISCUSSION TITLE
Basic Rate Limiting built-into the Epic Stack 🎉

### DISCUSSION THREAD

**Discussion by kentcdodds** [epicweb-dev/epic-stack#381] [Announcements]
You'll find the decision document here: https://github.com/epicweb-dev/epic-stack/blob/main/docs/decisions/025-rate-limiting.md

And the commit here: https://github.com/epicweb-dev/epic-stack/commit/6dd8e818bd378edd7075dda7c14bdb1ace801a6e

Cheers!

**Comment by vinstah** [comment]
Awesome Work, jut wondering if you are going through PR? I know you are busy but changes can be talked about in PRs

**Reply by kentcdodds** [reply]
I intentionally don't go through PRs. I definitely welcome feedback and ideas, but this project is my own opinions and decisions. There are some places that are good to have a committee of folks working on the solution together, but we're not going to do that here :)
